pkg/collector/corechecks/net/networkv2/network.go

```diff
@@ -456,6 +497,14 @@ func getEthtoolMetrics(driverName string, statsMap map[string]uint64) map[string
 			}
 		}
 		if newKey != "" && queueTag != "" && metricPrefix != "" {
+			if queueTag != "global" {
+				// we already guard against parsing unsupported NICs
+				queueMetrics := ethtoolMetricNames[driverName]
+				// skip queues metrics we don't support for the NIC
+				if !slices.Contains(queueMetrics, newKey) {
```

Comment:
we were only filtering [global metrics](https://github.com/DataDog/datadog-agent/pull/40326/files#diff-e88657054ac54654934fe01fd1722599b882419fb419c62b9d4a8c861ffe9dabR492) but not queue metrics, this led to many additional metrics that the python check does not submit